configs/projects/puppet-agent.rb

```diff
@@ -89,7 +89,6 @@
   proj.component "hiera"
   proj.component "leatherman"
   proj.component "cpp-hocon"
-  proj.component "marionette-collective"
```

Comment:
Not able to comment on the files lines here, but it looks like there are a few mcollective-specific project settings - e.g. `mco_root`, `mco_bindir`, `mco_libdir` - that we may also be able to remove?